src/config.ts

```diff
@@ -0,0 +1,16 @@
+import { readFileSync } from 'fs';
+import path from 'path';
+
+interface Config {
+    bot_token: string;
+    json_db_path: string;
+    sequelize: {
+        force: boolean;
+        alter: boolean;
+    }
+}
+
+const rawConfigData = readFileSync(path.join(process.cwd(), './config.json'), { encoding: 'utf-8' });
```

Comment:
Spaces -> tabs and a small styling change just to keep it inline with the rest of the codebase

```suggestion
interface Config {
	bot_token: string;
	json_db_path: string;
	sequelize: {
		force: boolean;
		alter: boolean;
	}
}

const rawConfigData = readFileSync(path.join(process.cwd(), './config.json'), {
	encoding: 'utf-8'
});
```

Reply:
Fixed in latest commit, I thought it had an editorconfig so I didnt pay attention to it

Reply:
That took care of the spaces but not the `rawConfigData` formatting

Reply:
Whoops, didnt notice that changed as well, fixed (I hope?)